guides/source/configuring.md

```diff
@@ -707,6 +707,8 @@ There are a few configuration options available in Active Support:
 
 * `config.active_support.use_sha1_digests` specifies whether to use SHA-1 instead of MD5 to generate non-sensitive digests, such as the ETag header. Defaults to false.
 
+* `config.active_support.use_authenticated_message_encryption` Use AES-256-GCM authenticated encryption as default cipher for encrypting messages instead of AES-256-CBC. Defaults to true.
```

Comment:
How about making it a sents, like the items around it?

Reply:
It's in the new defaults for rails 5.2. This is a copy and paste from the upgrade initializer generated with rails app:update.

Reply:
LGTM